internal/distro/rhel90/pipelines.go

```diff
@@ -56,14 +56,9 @@ func qcow2Pipelines(t *imageType, customizations *blueprint.Customizations, opti
 }
 
 func prependKernelCmdlineStage(pipeline *osbuild.Pipeline, t *imageType, pt *disk.PartitionTable) *osbuild.Pipeline {
-	if t.arch.name == s390xArchName {
```

Comment:
Is this also an issue for 8.5, is it tracked anywhere?

cc @achilleas-k

Reply:
Nah, this isn't an issue for 8.5. The change is in `/usr/lib/kernel/install.d/20-grub.install`, thus in `grub2-common`.